feat(tem): update quota information (#3848)
scaleway/docs-content: managed-services/transactional-email/reference-content/tem-capabilities-and-limits.mdx

  - managed-services
---

> Scaleway's Transactional Email Service allows users to send automated emails triggered by specific actions like account notifications, password resets, and order confirmations. This service is optimized for high-volume email sending, ensuring fast and reliable delivery of critical messages to end users.

Reviewer: ```suggestion
Scaleway's Transactional Email service allows users to send automated emails triggered by specific actions like account notifications, password resets, and order confirmations. This service is optimized for high-volume email sending, ensuring fast and reliable delivery of critical messages to end users.
```

The service is built on scalable infrastructure capable of handling substantial email volumes. However, sending capacities are evaluated on a case-by-case basis, depending on factors such as:

- Usage patterns
- Sender reputation
- Adherence to best practices

> New users may face stricter initial limits, while well-known customers can benefit from higher limits, potentially sending up to millions of emails. These quotas are subject to validation by support.

Reviewer: ```suggestion
New users may face stricter initial limits, while well-known customers can benefit from higher limits, potentially sending up to millions of emails. These quotas are subject to validation by the Scaleway support team.
```

<Message type="important">

>   Customers must adhere to industry standards and our [acceptable use policy](https://www.scaleway.com/en/terms/), including compliance with [Scaleway's anti-spam policy](https://tem.s3.fr-par.scw.cloud/antispam_policy.pdf). Failure to comply may result in temporary or permanent restrictions to the service.

Author: ```suggestion
  Customers must adhere to industry standards and our [General Terms of Services](https://www-uploads.scaleway.com/General_Terms_of_Services_v17072024_45d4879c08.pdf), including compliance with [Scaleway's anti-spam policy](https://tem.s3.fr-par.scw.cloud/antispam_policy.pdf). Failure to comply may result in temporary or permanent restrictions to the service.
```
